ComponentLib/PowerElectronics/InductionMotor/InductionMotor.cpp

```diff
@@ -20,20 +18,15 @@ namespace ModelLib {
  */
 
 template <class ScalarT, typename IdxT>
-InductionMotor<ScalarT, IdxT>::InductionMotor(IdxT id, ScalarT Lls, ScalarT Rs, ScalarT Llr, ScalarT Rr, ScalarT Lms, ScalarT RJ, ScalarT P)
-  : Lls_(Lls),
-    Rs_(Rs),
-    Llr_(Llr),
-    Rr_(Rr),
-    Lms_(Lms),
-    RJ_(RJ),
-    P_(P)
+InductionMotor<ScalarT, IdxT>::InductionMotor(
+    IdxT id, ScalarT Lls, ScalarT Rs, ScalarT Llr, ScalarT Rr, ScalarT Lms, ScalarT RJ, ScalarT P)
+    : Lls_(Lls), Rs_(Rs), Llr_(Llr), Rr_(Rr), Lms_(Lms), RJ_(RJ), P_(P)
```

Comment:
Can we configure formatting of functions with many arguments and initialization in constructor like below? It spans more lines of code but it is clearer and let you add comments for each variable, if necessary.
```suggestion
InductionMotor<ScalarT, IdxT>::InductionMotor(IdxT id, // explanation what this variable is
                                              ScalarT Lls, 
                                              ScalarT Rs, 
                                              ScalarT Llr, 
                                              ScalarT Rr, 
                                              ScalarT Lms, 
                                              ScalarT RJ, 
                                              ScalarT P)
    : Lls_(Lls), 
      Rs_(Rs), 
      Llr_(Llr), 
      Rr_(Rr), 
      Lms_(Lms), 
      RJ_(RJ), 
      P_(P)
```

Reply:
It is already set such that If you were to add a comment to one of the lines as shown, then the formatter will format as shown.